distributed/actor.py

```diff
@@ -118,13 +119,27 @@ def __dir__(self):
         return sorted(o)
 
     def __getattr__(self, key):
-        attr = getattr(self._cls, key)
+        print("###getattr", file=open("temp", "at"))
 
         if self._future and self._future.status not in ("finished", "pending"):
             raise ValueError(
                 "Worker holding Actor was lost.  Status: " + self._future.status
             )
 
+        if self._worker and self._worker.address == self._address:
+            actor = self._worker.actors[self.key]
+            attr = getattr(actor, key)
+
+            if iscoroutinefunction(attr):
+                return lambda *args, **kwargs: attr(*args, **kwargs)
```

Comment:
I'm confused by this.   It seems like a no-op?  Would `return attr` be equivalent?

Reply:
That seems ... likely :). It must have been a little more complex as I tried&errored here.